src/main/java/org/commcare/formplayer/aspects/MetricsAspect.java

```diff
@@ -118,9 +125,20 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
             );
         }
 
-        if (timer.durationInMs() >= 60 * 1000) {
-            sendTimingWarningToSentry(timer);
+        long extremelySlowRequestThreshold = 60 * 1000;
+        Map<String, Long> slowRequestThresholds = getSlowRequestThresholds();
+
+        if (timer.durationInMs() >= extremelySlowRequestThreshold) {
+            sendTimingWarningToSentry(timer, EXTREMELY_SLOW_REQUEST);
+        } else if (slowRequestThresholds.containsKey(requestPath) && timer.durationInMs() >= slowRequestThresholds.get(requestPath)) {
+            // limit slow requests sent to sentry, send 1 for every 100 requests
+            int chanceOfSending = 100;
```

Comment:
It might be nice to start this at 1000. We can always increase if necessary, but it will prevent us from hitting any limits

Reply:
Changed to 1000.